### DISCUSSION TITLE
DDF output for data augmentaiton transforms

### DISCUSSION THREAD

**Discussion by mariakawula** [Project-MONAI/tutorials#600] [Q&A]
Hi,

is there a way to output the dense displacement field that was used to deform images for data augmentation? I defined a Compose transform that contains, among others, RandAffine and BSpline based deformation field (RandomElasticDeformation from torchio).

I can extract the affine matrix that in principle could be used to get the ddf but I don't have anything for B-splines.

**Comment by yiheng-wang-nv** [comment]
Hi @mariakawula , could you post your Compose transform here, thus I'm able to have a try.

**Reply by yiheng-wang-nv** [reply]
duplicate to https://github.com/Project-MONAI/MONAI/discussions/3933
